### PR TITLE
[3500] An item is being moved from one shelf to another (folijet) (TaaS)

### PR DESCRIPTION
**Pull request by dnovikau**
[C3500](https://foliotest.testrail.io/index.php?/cases/view/3500)
***
[Allure report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5067/allure)
***

### CODE REVIEW

**Review by ernestyedigaryan1 — APPROVED** [review]

File: cypress/e2e/inventory/cataloging/item-moved-from-one-shelf-to-another.cy.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import { Locations, ServicePoints } from '../../../support/fragments/settings/tenant';
+import { randomFourDigitNumber } from '../../../support/utils/stringTools';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import Users from '../../../support/fragments/users/users';
+
+describe('Inventory', () => {
+  describe('Cataloging', () => {
```
Comment (TetianaParanich) [inline cypress/e2e/inventory/cataloging/item-moved-from-one-shelf-to-another.cy.js:10]: Let's use 2 describes like
describe('inventory', () => {
  describe('Cataloging -> Maintaining the catalog', () => {

File: cypress/e2e/inventory/cataloging/item-moved-from-one-shelf-to-another.cy.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          const HoldingsRecordView = InventoryInstance.openHoldingView();
+
+          // Click on "Actions" menu, Select "Edit"
+          const HoldingsRecordEdit = HoldingsRecordView.edit();
+
+          // Change the Call number -> Click "Save & Close" button
+          HoldingsRecordEdit.fillCallNumber(testData.callNumber);
+          HoldingsRecordEdit.saveAndClose({ holdingSaved: true });
+
+          // Test is failing because of an Issue:
```
Comment (TetianaParanich) [inline cypress/e2e/inventory/cataloging/item-moved-from-one-shelf-to-another.cy.js:72]: Let's add the comments about the bug report under the it block

Comment (dnovikau) [reply cypress/e2e/inventory/cataloging/item-moved-from-one-shelf-to-another.cy.js]: Done

File: cypress/e2e/inventory/cataloging/item-moved-from-one-shelf-to-another.cy.js (hunk 1 above)
Comment (dnovikau) [reply cypress/e2e/inventory/cataloging/item-moved-from-one-shelf-to-another.cy.js]: Done

**Pull request merged**